.gitignore

```diff
@@ -1,42 +1,42 @@
-# Virtualenv
-ENV
-env
-.Python
-include
-lib
-
-# Python Packages
-*.egg
-*.egg-info
-bin
-build
-dist
-sdist
-eggs
-env
-parts
-var
-develop-eggs
-.installed.cfg
-
-docs/.build
-docs/_build
-
-# Pip Installer Log
-pip-log.txt
-
-# Unit test / coverage
-.coverage
-.tox
-
-# Mac OS X
-.DS_Store
-
-# Generated files
-*.[oa]
-*.py[co]
-*.rbc
-
-#Temporary Files
-*.py~
+# Virtualenv
+ENV
+env
+.Python
+include
+lib
+
+# Python Packages
+*.egg
+*.egg-info
+bin
+build
+dist
+sdist
+eggs
+env
+parts
+var
+develop-eggs
+.installed.cfg
+
+docs/.build
+docs/_build
+
+# Pip Installer Log
+pip-log.txt
+
+# Unit test / coverage
+.coverage
+.tox
+
+# Mac OS X
+.DS_Store
+
+# Generated files
+*.[oa]
+*.py[co]
+*.rbc
+
+#Temporary Files
+*.py~
```

Comment:
My git configuration converted the files to LF automatically and since most of the other files were LF format I thought I'd commit it as a change so they were all the same format. Doesn't matter really although most people prefer LF.